README.md

```diff
@@ -35,7 +31,7 @@ received content.
 
 ## Goals
 
-Notary is based on [The Update Framework](https://www.theupdateframework.com/), a secure general design for the problem of software distribution and updates. By using TUF, Notary achieves a number of key advantages:
+The notary client and server is based on [The Update Framework](https://www.theupdateframework.com/), a secure general design for the problem of software distribution and updates. By using TUF, Notary achieves a number of key advantages:
```

Comment:
Should we use `notary` to make sure we mean the repository? Maybe change to "The `notary` client and server implement TUF"?